data/kiwi/appliance.kiwi

```diff
@@ -0,0 +1,391 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-Profiles: @BUILD_FLAVOR@ -->
+<!-- OBS-Milestone: %current_milestone -->
+
+<image schemaversion="6.1" name="SLES15-SP4-Minimal" displayname="SLES 15 SP4">
+    <description type="system">
+        <author>SUSE LLC</author>
+        <contact>crc@suse.com</contact>
+        <specification>SUSE Linux Enterprise 15 SP4 Minimal</specification>
+    </description>
+    <profiles>
+        <profile name="kvm-and-xen" description="Minimal for KVM and Xen" arch="x86_64"/>
+        <profile name="kvm" description="Minimal for KVM" arch="aarch64"/>
+        <profile name="VMware" description="Minimal for VMware" arch="x86_64"/>
+        <profile name="MS-HyperV" description="Minimal for Hyper-V" arch="x86_64"/>
+        <profile name="OpenStack-Cloud" description="Minimal for OpenStack Cloud" arch="x86_64"/>
+        <profile name="RaspberryPi" description="Minimal for the Raspberry Pi" arch="aarch64"/>
+    </profiles>
+    <preferences profiles="kvm-and-xen">
+        <version>15.4.0</version>
```

Comment:
I think version here shouldn't be hardcoded

Reply:
According to the Kiwi documentation, the mandatory image version should follow a three-part format: Major.Minor.Release. Therefore, I have modified the version to 1.1.0 in the Kiwi XML file. After the Kiwi build is completed,I rename the version in the generated qcow2 file to match the build setting value.This step enables me to further test the generated qcow2.